Jenkinsfile

```diff
@@ -67,7 +67,13 @@ pipeline {
       steps {
         sh '''#!/bin/bash
           set -exo pipefail
-          docker image prune -f # remove previously built image to prevent disk from filling up
+          # Remove images (dangling or not) created more than 120h (5 days ago) to prevent disk from filling up.
+          docker image prune --all --force --filter "until=120h" --filter "label=kaggle-lang=r"
```

Comment:
Should we add the label to the rstats images too?

Reply:
The `rstats` image inherits the label from its parent (i.e. `rcran`). The labels were added here: https://github.com/Kaggle/docker-rcran/pull/28